test/functional/test_framework/wallet.py

```diff
@@ -77,3 +77,30 @@ def send_self_transfer(self, *, fee_rate=Decimal("0.003"), from_node, utxo_to_sp
         assert_equal(tx_info['vsize'], vsize)
         assert_equal(tx_info['fee'], fee)
         return {'txid': txid, 'wtxid': tx_info['wtxid'], 'hex': tx_hex}
+
+    def send_multi_self_transfer(self, *, fee_rate=Decimal("0.03"), from_node, utxo_to_spend=None, big_txouts=None):
```

Comment:
I'm not sure this method name is ideal? I was expecting multiple txns, but I guess this sends one big transaction? I'm not sure `send_big_self_transfer` would be better, but just want to double check on naming here